server.json

```diff
@@ -1,9 +1,13 @@
 {
     "app":{
+        "cfengine":"lucee@5.4.3.2",
         "libDirs":"lib/java"
     },
     "open":true,
     "web":{
+        "http":{
```

Comment:
@michaelborn Would you updating your PR to remove the port definition so that the server will start on a random port? Thanks

Reply:
Man, sorry I missed this!